package.json

```diff
@@ -114,6 +114,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^8.0.2",
     "jest": "^27.5.1",
+    "jest-each": "^29.3.1",
```

Comment:
I think this is fine to add, but it should match our current jest number - since we haven't upgraded TS yet we can't upgrade jest past 27. Can you downgrade this to 27.x?

Reply:
Sure. I have downgraded the package.